samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,284 @@
+   +------------------+---------------+-----------------------------+
+   | Length           | 20 00         | 0x0020                      |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 03            | Message                     |
+   +------------------+---------------+-----------------------------+
+   | Length           | 20 00         | 0x0020                      |
+   +------------------+---------------+-----------------------------+
+
+Note that the example application will request all existing attribute types, not only a subset.
+
+The following table shows the format of a response that contains some of the requested notification attributes:
+
+   +------------------+---------------+-----------------------------+
+   | Field            | Example value | Interpretation              |
+   +==================+===============+=============================+
+   | Command ID       | 0             | Get notification attributes |
+   +------------------+---------------+-----------------------------+
+   | Notification UID | 01 02 03 04   | 67305985 (0x4030201)        |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 01            | Title                       |
+   +------------------+---------------+-----------------------------+
+   | Length           | 03 00         | 0x0003                      |
+   +------------------+---------------+-----------------------------+
+   | Data             | 6E 52 46      | "nRF"                       |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 03            | Message                     |
+   +------------------+---------------+-----------------------------+
+   | Length           | 02 00         | 0x0002                      |
+   +------------------+---------------+-----------------------------+
+   | Data             | 35 32         | "52"                        |
+   +------------------+---------------+-----------------------------+
+   | Attribute ID     | 00            | App identifier              |
+   +------------------+---------------+-----------------------------+
+   | Length           | 03 00         | 0x0003                      |
+   +------------------+---------------+-----------------------------+
+   | Data             | 63 6F 6D      | "com"                       |
+   +------------------+---------------+-----------------------------+
+
+1. Press Button 1 to request notification attributes for the iOS notification that was received.
+#. In nRF Connect, verify that the ANCS Control Point (0xD8F3) is updated to ``00 01 02 03 04 00 01 20 00 02 20 00 03 20 00 04 05 06 07``.
+#. Respond to the request by sending two notification attributes: the title and the message. To do this, first set the Data Source (characteristic 0xC6E9) in the Server to ``00 01 02 03 04 01 03 00 6E 52 46 03 02 00 35 32``.
```

Comment:
```suggestion
#. Respond to the request by sending two notification attributes: the title and the message. Set the Data Source (characteristic 0xC6E9) in the Server to ``00 01 02 03 04 01 03 00 6E 52 46 03 02 00 35 32``.
```

Reply:
Done.

Besides, Data Source is modified to a GUI label 'Apple Data Source'. It aligns with nRF Connect for Desktop.